recipe/meta.yaml

```diff
@@ -29,6 +29,7 @@ requirements:
         - numpy x.x
         - cairo
         - pango
+        - pyqt
```

Comment:
@karlmsmith I forgot about this dependency before! I always had it pre-installed and never notice that we must specify it here to get the graphs to display properly. Let me know if you can spot any other missing dependency that I should add.